Go/Symbols/GoConstSymbols.tmPreferences

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
You can define both, the indexed and non-indexed symbols, in the same file, assuming their scope selectors are equal. This applies to the other files as well.

Reply:
These are defined in separate files to keep `symbolTransformation` from applying to the indexed symbols and messing up "goto_definition". I'd love to be able to merge the files, but it doesn't seem possible while keeping the current rule logic.

Reply:
Can you confirm that `symbolTransformation` does indeed affect the symbol that is used for the index if `showInIndexedSymbolList` is set? Previous experience doesn't indicate so (and neither to [the docs](http://docs.sublimetext.info/en/latest/reference/symbols.html) derived from that).

Even with two different files, since the scope selectors are the same it would have the exact same effect as all the meta settings will be set on any token that is matched.

Reply:
Interesting! I double checked, and indeed, it looks like `symbolTransformation` doesn't affect indexing. Must have mixed it up with something else when I was writing this.

Merged some of the symbol files. Thanks!